base/Dockerfile

```diff
@@ -47,7 +49,11 @@ RUN apt-get update && \
     rm -rf /var/cache/apt /var/lib/apt/lists/*; \
     # install apt key if supported kopano
     if [ ${RELEASE_KEY_DOWNLOAD} -eq 1 ]; then \
-        curl -s -S -o - "${KOPANO_CORE_REPOSITORY_URL}/Release.key" | apt-key add -; \
+        if [ "$(awk 'BEGIN { FS="[ ]" } ; { print $6 }' < /etc/apt/auth.conf.d/kopano.conf)" != "REPLACE-ME" ]; then \
```

Comment:
Maybe it would be easier to just add the key to this repo and skip pulling it at build?

Reply:
always add it? or only if ${RELEASE_KEY_DOWNLOAD} is true?

Reply:
I'd say always add. I don't think it will hurt if its there if you don't use it.

Maybe the RELEASE_KEY_DOWNLOAD env can go in general? (but that could still be a followup and does not need to be handled in this pr)